common-pg/src/main/scala/com/socrata/pg/store/SecondarySchemaLoader.scala

```diff
@@ -48,22 +50,61 @@ class SecondarySchemaLoader[CT, CV](conn: Connection, dsLogger: Logger[CT, CV],
   }
 
   def createFullTextSearchIndex(columnInfos: Iterable[ColumnInfo[CT]]): Unit = {
+    // Bit of a dance here because we only want to create the index if
+    // it isn't already what it's supposed to be, and "what it's
+    // supposed to be" is more complicated than for individual column
+    // indices (e.g., if a new column is created, this index needs to
+    // be recreated if it already exists).  In order to find out if we
+    // need to do this, we'll use postgresql's comment facility to tag
+    // the index with a hash of the SQL we're using to define the
+    // index.
+    //
+    // Postgresql does give a way to get SQL for the index in the
+    // pg_indexes table, but it's beenñ normalized and it would
```

Comment:
`beenñ`

Reply:
Isn't that how one conjugatedñ english past-tense verbs?

Reply:
No it isn'tñ.